src/sentry/api/serializers/models/group.py

```diff
@@ -141,8 +141,9 @@ def get_attrs(self, item_list, user):
             ).select_related('user')
         )
 
-        user_counts = tagstore.get_group_values_seen(
-            [g.id for g in item_list], environment_id=None, key='sentry:user')
+        # TODO(brett): support environment in this serializer
+        user_counts = tagstore.get_groups_user_counts(
+            item_list[0].project_id, [g.id for g in item_list], environment_id=None)
```

Comment:
Probably a reasonable assumption, but we're potentially changing the behavior if, somehow, a serializer for multipel groups spread across multiple different projects, would break. Maybe we should add some assertions when we do stuff like this to assert all of `item_list[*].project_id` are the same and be explicit about this behavior of not allowing cross project stuff. (This has come up elsewhere where we make a similar assumption)

Reply:
That's a big reason I punted on that change in this PR. I honestly have no idea if we have an org-wide view (or API endpoint) of groups. I actually think we might already. I'll add this to my notes.